admin/class-yoast-plugin-conflict.php

```diff
@@ -311,4 +319,15 @@ private function remove_deactivated_plugin() {
 			unset( $this->all_active_plugins[ $key_to_remove ] );
 		}
 	}
+
+	/**
+	 * Get the identifier from the plugin file
+	 *
+	 * @param string $plugin_file Plugin file to get Identifier from.
+	 *
+	 * @return string
+	 */
+	private function get_notification_identifier( $plugin_file ) {
+		return md5($plugin_file);
```

Comment:
Mainly because the `$plugin_file` will hold a very long string i.e.
"facebook-and-digg-thumbnail-generator/facebook-and-digg-thumbnail-generator.php"
or
"facebook-featured-image-and-open-graph-meta-tags/fb-featured-image.php"
Which would have the unique folder name to use; but still hold an "_insane_" amount of characters.

This keeps everything in more manageable ranges.